projects/mac_ai/testing/prepare_llama_cpp.py

```
@@ -46,7 +47,7 @@ def retrieve(prefix):
 
         try:
             version = remote_access.read(llama_latest_file).strip()
-        except Exception:
+        except subprocess.CalledProcessError:
             msg = "Couldn't fetch the llama.cpp latest version identifier from the system under test. Prepare it first"
             logging.error(msg)
```

Ruff (0.13.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)